package.json

```diff
@@ -31,10 +31,12 @@
   },
   "dependencies": {
     "body": "^5.0.0",
+    "commander": "^2.6.0",
```

Comment:
IIRC `main.js` is used only during development (run it manually), so I think this doesn't need to be a binary.

+1 for making commander a devDependency.